ignite/cmd/cmd.go

```diff
@@ -161,6 +162,18 @@ func flagGetClearCache(cmd *cobra.Command) bool {
 	return clearCache
 }
 
+func flagSetPluginsGlobal() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.BoolP(flagPluginsGlobal, "g", false, "use global plugins configuration"+
+		" ($HOME/.ignite/plugins/plugins.yml)")
+	return fs
+}
+
+func flagGetPluginsGlobal(cmd *cobra.Command) bool {
+	global, _ := cmd.Flags().GetBool(flagPluginsGlobal)
+	return global
+}
```

Comment:
this should go to `cmd/plugin.go`, as they are only related to this namespace right ?

Reply:
https://github.com/ignite/cli/pull/3214/commits/0bea0598b767d61ddb70626b4d60742df843771c